### ISSUE TITLE
precision plot issue

### ISSUE THREAD

**Issue by Aksen86** [OskarHansson/strvalidator#23]
Dear Oskar

We had problem with plot generating by strvalidator generally. We installed older one version of R and it've helped. We started to generate plots but not in preision section. We can generate statistic in this section but not graphic plot. unfortuanatelly we cant install older one strvalidator f.e.g 2.2.0 
Do you know what could be the solution for that issue?

Kind regards
Aga Smolen

**Comment by OskarHansson** [comment]
Hi Aga,

I have tried a bugfix after which I can plot precision plots from the ISFG 2022 workshop using R version 4.3.0 and ggplot2_3.4.2. The fix is only implemented for plotPrecision_gui and I have not done extensive testing. Please feel free to install and test. Give me some feedback on how it works and details about the dataset and steps performed.

**To install from GitHub**
i. Install the devtools package from CRAN by typing this in the Console window (press Enter to execute the command):

`install.packages("devtools", dependencies = TRUE)`   

ii. Load the devtools package and install the strvalidator package from GitHub:

```
library(devtools)

install_github("OskarHansson/strvalidator")
```

**Comment by OskarHansson** [comment]
Should be fixed in version 2.4.1, which will be released to CRAN within a few days.
